src/fields/fields.service.ts

```
@@ -340,14 +340,38 @@ export class FieldsService {
-      }
+      // Step 4: Get field type from schema - check both direct type and schema.type
+      const fieldType = fieldMetadata.schema?.type || fieldMetadata.type; // Type from schema: "boolean", "string", "array"
+      // Step 5: Handle comma-separated values (current workflow)
+      // Only split if the field type in schema is 'array' and contains comma
+      // Also check if this is actually a multi-select field by looking at fieldParams
+      const isMultiSelect =
+        fieldMetadata.fieldParams?.options?.enum &&
+        Array.isArray(fieldMetadata.fieldParams.options.enum);
+
+      if (
+        fieldType === 'array' &&
```

ESLint

[error] Replace `'array'` with `"array"`

(prettier/prettier)